### DISCUSSION TITLE
[Enhancement] [table] table组件新增renderCell属性

### DISCUSSION THREAD

**Discussion by SorrowX** [element-plus/element-plus#17417] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `table`

## Description

1. 希望增加 renderCell属性，用于编写render函数来渲染table单元格内容
2. 这个属性其实在element-ui/plus，本身就有，只是没有放开，从renderHeader的警告，可以看出，官方更推荐使用模板而不是render, 但有时配置化时，render更灵活。所以希望能放开renderCell属性

### Additional comments

_(empty)_

**Comment by GauharChan** [comment]
@jw-foss
